### PR TITLE
HW#5 - B. Young

### PR DESCRIPTION
**Pull request by bscottyoung**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+
+20.times do |n|
+    a = Article.new
+    a.author_first_name =  Faker::Name.first_name
```
Comment (kareemgrant) [inline db/seeds.rb]: @bscottyoung nice use of the faker gem

**Comment by kareemgrant** [comment]
@bscottyoung There's something strange happening with your git process. There should not be 53 **Files Changed**. The only files changed should be the files that were changed during your implementation of Task #1.

screen shot 2016-03-07 at 12 06 23 am

**Comment by bscottyoung** [comment]
Hmmm.  I changed the directory it was in.  Let me fix that.

Brian Young
404.229.6006

**Comment by kareemgrant** [comment]
@bscottyoung this still has not been fixed.

**Comment by bscottyoung** [comment]
I don't understand what "this" is.  Please specify.  If the multiple files...I tried to change with Brian with no success.

**Comment by kareemgrant** [comment]
@bscottyoung "this" refers to the issue of having 53 changed files. I'll help you resubmit correctly in class today

**Pull request closed without merging**